ethereum/contracts/pyth/Pyth.sol

```diff
@@ -223,19 +223,22 @@ abstract contract Pyth is PythGetters, PythSetters, AbstractPyth {
 
         // Check that there is not a significant difference between this chain's time
         // and the price publish time.
-        if (diff(block.timestamp, info.priceFeed.publishTime) > VALID_TIME_PERIOD_SECS) {
+        if (info.priceFeed.status == PythStructs.PriceStatus.TRADING && 
```

Comment:
maybe leave a comment explaining this logic

Reply:
Done.